app/assets/stylesheets/templates/_messages.scss

```diff
@@ -28,23 +26,27 @@
       height: auto;
       max-width: 100%;
     }
-
+    
     .message--outbound {
-      flex-basis: 55%;
       margin-left: auto;
 
       .message--content {
+        max-width: 100%;
         background: $color-teal;
-        border-radius: 8px;
+        border-radius: 10px 10px 0px 10px;
         color: $color-background;
         margin-bottom: 10px;
         padding: {
           bottom: .5em;
-          left: $padding-small;
-          right: $padding-small;
+          left: 1.5em;
+          right: 1.5em;
```

Comment:
Could you assign a variable for this value (`1.5em` or an equivalent in `rem`) in `app/assets/stylesheets/atoms/_variables.scss` and use it above, and on lines 74-75?